npc/cities/umbala.txt

```diff
@@ -1021,7 +1021,7 @@ umbala,140,197,1	script	Bungee Jump	FAKE_NPC,0,0,{
 OnTouch:
 	switch(rand(1,3)) {
 	case 1:
-		percentheal -100,0;
+		Hp -= Hp-1;
```

Comment:
How about using `Hp = 1` since you reduce the Hp to 1 only ?

It would avoid the "calculation" part for `Hp - 1` and more simple to understand for those who are not familiar with coding.